makefile

```diff
@@ -0,0 +1,12 @@
+
+docs-html:
+	docker run -it --rm -v $(shell pwd):/docs sphinxdoc/sphinx:7.3.6 bash -c "cd doc/sphinx-guides && pip3 install -r requirements.txt && make clean && make html"
```

Comment:
Can we factor out 7.3.6 to a variable? Or better yet, get it from `doc/sphinx-guides/requirements.txt`? It should actually be 7.2.6.

Reply:
Done @pdurbin 😄